src/js/product-data.js

```diff
@@ -0,0 +1,80 @@
+// Dat IIFE tho...
+;(function(globals){
+
+angular.module('data-fetch',[])
+
+  .run(['$rootScope', function($scope){
+    $scope.count = '249';
+    $scope.shop_name = 'PickslaysWoodworking';
+  }])
+
+  .run([ '$rootScope','$http', function($scope, $http){
+    $http.get("../../apis/etsy/listings/productpage/item100384851_images.json") // will work running html in Browser Sync
+      .then(function(response){
+        console.log(response);
+        $scope.listingImages = response.data;
+
+        var counter = 0;
+        $('a#next').on('click', function () {
+          if (counter >= 0 && counter <= 3 ) {
+          counter++;
+          $('img.img-active').attr("src",response.data.results[counter].url_570xN);
+        }
+        });
```

Comment:
i inserted the event handlers inside the `then`  because the identifier `response.data` is local within  `then`.... maybe I can assign `resposnse.data` to a pre-declared global variable then do the event handlings globally. looking forward to refactor this together...